airflow/contrib/operators/telegram_operator.py

```diff
@@ -0,0 +1,89 @@
+import json
+import telegram
+
+from airflow.models import BaseOperator
+from airflow.utils.decorators import apply_defaults
+from airflow.exceptions import AirflowException
+from hooks.telegram_hook import TelegramHook
+
+
+class TelegramAPIOperator(BaseOperator):
+    """
+    Base Telegram Operator
+    The TelegramAPIPostOperator is derived from this operator.
+    In the future additional Telegram bot API Operators will be derived from this class as well
+    :param telegram_conn_id: Telegram connection ID which its password is Telegram API token
+    :type telegram_conn_id: str
+    :param token: Telergram bot API token
+    :type token: str
+    """
+
+    @apply_defaults
+    def __init__(self, telegram_conn_id=None, token=None, *args, **kwargs):
```

Comment:
```suggestion
    def __init__(self, telegram_conn_id=None, token=None, *args, **kwargs):
```
Keeping passwords in other places is a security risk.

Reply:
Related discussion: https://github.com/apache/airflow/pull/4895#issuecomment-471385356